source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -9,43 +9,29 @@ parent: "/manual.html"
-> postcode lookups will not work when using the secondary CDN.
+> **Important** The failover CloudFront distribution does not have feature parity with the primary Fastly service.
+> Most features will continue to work, including search, smart answers, and postcode lookups. Some features, including
+> A/B testing, will not work.
 
 ## Fail over checklist
 
 - Confirm that Fastly is the cause of the incident (check [https://status.fastly.com/](https://status.fastly.com/)
   and keep an eye on twitter - if there's a major Fastly outage there will be a lot of noise)
+- Escalate to GOV.UK SMT as soon as you begin to consider failing over
 - Sign in to the AWS console as an admin (`gds aws govuk-production-admin -l`, or however you prefer to sign in to AWS)
 - Sign in to [the GCP console](https://console.cloud.google.com/home/dashboard?project=govuk-production)
 - Open the following four pages as separate tabs:
   - [GCP Cloud DNS www-cdn.production.govuk.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/govuk-service-gov-uk/rrsets/www-cdn.production.govuk.service.gov.uk./CNAME/edit?project=govuk-production)
   - [AWS Route 53 govuk.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z22RPYZA77J620)
   - [GCP Cloud DNS assets.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/assets.publishing.service.gov.uk./CNAME/edit?project=govuk-production)
   - [AWS Route 53 publishing.service.gov.uk](https://console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z3SBFBO09PD5HF)
-- Because the secondary CDN provides degraded service we have a grace period
-  before failing over
-  - Escalate to GOV.UK SMT as soon as you begin to consider failing over
-  - For a major Fastly outage (more than 30% of requests failing) wait 15
-    minutes (from the start of the incident) and then start the fail over
-  - For a minor Fastly outage (less than 30% of requests failing) wait at least
-    15 minutes (from the start of the incident) before failing over - discuss
-    the decision with the GOV.UK SMT Escalations person
-- While you wait, confirm the new CNAMES for www-cdn.production.govuk.service.gov.uk and assets.publishing.service.gov.uk
-  - This [Draft PR to Failover to AWS CloudFront](https://github.com/alphagov/govuk-dns-config/pull/714) shows the CNAMES you need to change, and how to test that they are correct
-  - You can also get the CNAMEs to use for the secondary CDN from the AWS CLI: <div class="highlight"><pre class="highlight plaintext"><code># www.gov.uk
-  gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='WWW'].DomainName | [0]"
-  # assets.publishing.service.gov.uk
-  gds aws govuk-production-readonly aws cloudfront list-distributions --query "DistributionList.Items[?Comment=='Assets'].DomainName | [0]"</code></pre></div>
```

Comment:
These snippets look useful to copy into the PR description for govuk-dns-tf#69.

Reply:
Ah yes. They'll be [removed soon](https://trello.com/c/vhKRQfLu/3296-delete-the-old-static-cloudfront-distributions), but we do want a working snippet in the meantime (or no snippet at all).